tstdc/sys/Cargo.toml

```diff
@@ -18,10 +18,10 @@ test = false
 doctest = false
 
 [dependencies]
-mc-sgx-core-sys-types = { path = "../../core/sys/types", version = "0.7.1" }
-mc-sgx-tstdc-sys-types = { path = "types", version = "0.7.1" }
+mc-sgx-core-sys-types = { path = "../../core/sys/types", version = "=0.7.2" }
+mc-sgx-tstdc-sys-types = { path = "types", version = "=0.7.2" }
 
 [build-dependencies]
 bindgen = "0.66.1"
 cargo-emit = "0.2.1"
-mc-sgx-core-build = { path = "../../core/build", version = "0.7.1" }
+mc-sgx-core-build = { path = "../../core/build", version = "=0.7.2" }
```

Comment:
Notice that these went from "0.7.1" to "=0.7.2".
The lack of "=" was causing cargo release to not update these, since "0.7.2" means 0.7.2 or better, while "=0.7.2" means exactly 0.7.2